torch/nn/modules/loss.py

```diff
@@ -458,9 +511,23 @@ class HingeEmbeddingLoss(_Loss):
     dissimilar, e.g. using the L1 pairwise distance as `x`, and is typically
     used for learning nonlinear embeddings or semi-supervised learning::
 
-                         { x_i,                  if y_i ==  1
-        loss(x, y) = 1/n {
-                         { max(0, margin - x_i), if y_i == -1
+    The loss function for :math:`n`-th sample in the mini-batch is:
+
+    .. math::
+        l_n = \begin{cases}
+        x_n, & \text{if}\; y_n = 1,\\
+        \max \{0, \Delta - x_n\}, & \text{if}\; y_n = -1,
```

Comment:
can you please align the cases? this is useful when viewing the documentation inline in the shell